python/cudf/cudf/_lib/csv.pyx

```diff
@@ -9,6 +9,7 @@ from libcpp.vector cimport vector
 
 cimport cudf._lib.cpp.types as libcudf_types
 from cudf._lib.cpp.types cimport data_type, type_id
+from cudf._lib.io.datasource cimport Datasource, NativeFileDatasource
```

Comment:
Since we're doing `from ... import Datasource`, we could replace `cudf._lib.io.datasource.Datasource` a few lines down. Or we could remove this import:

```suggestion
from cudf._lib.io.datasource cimport NativeFileDatasource
```

Reply:
Good call - I went with the `cudf._lib.io.datasource.Datasource` -> `Datasource` change below